vyper/utils.py

```diff
@@ -591,3 +592,12 @@ def annotate_source_code(
     cleanup_lines += [""] * (num_lines - len(cleanup_lines))
 
     return "\n".join(cleanup_lines)
+
+
+def safe_relpath(path):
+    try:
+        return os.path.relpath(path)
+    except ValueError:
+        # on Windows, if path and curdir are on different drives, an exception
```

Comment:
Would the path be an absolute path in this scenario?

Reply:
yeah, I'm not sure about this

the reported issue targeted builtins, which use `__path__`, which is absolute (and thus, the approach should work). although `safe_relpath` which returns an abs path is a bit weird

but it might be that a relative path is in search paths, right? And that path could reference a different drive causing the exception. but in that case it's likely a user error?

Reply:
relpath is also used even if a different search path is used to resolve a path, for instance for error reporting it is used to report relative to the user's current directory. but on windows if you can't construct the relpath, you can't construct the relpath